lib/common.js

```diff
@@ -94,3 +95,37 @@ module.exports.baseHandler = ({ command, formatter }) => async ({
     log(err, { severity: 'error' })
   }
 }
+
+const versionPattern = /([0-9]\d*\.[0-9]\d*\.[0-9]\d*)(-alpha||-beta)?(-[0-9]\d*)?/
+
+module.exports.tagHandler = (command) => async ({ github, verbose }) => {
+  const _exec = exec({ verbose })
+  try {
+  // array of the top 3 commits in master
+    const commits = _exec("git log --no-merges --format='%s' -n 3").toString().trim().split('\n')
```

Comment:
I think it is still safe to check for a few commits, let's say the workflow check passed as a false positive, for those PRs who have the version in a commit and then another commit with `version - fix typo in version` we could still take advantage of this automation. And if there is no tag in the top 3 commits, it would exit with a message `No version bump commit available`

Reply:
🤔 ok then let's open the number of backtracing commits an option, which defaults to 3